File: app/src/main/java/in/encrust/writersden/FontModel.java
package in.encrust.writersden;

import android.graphics.Typeface;

public class FontModel {
    private Typeface customfont;
    private String fontname;

    public FontModel() {
    }

    public FontModel(Typeface customfont, String fontname) {
        this.customfont = customfont;
        this.fontname = fontname;
    }

    public Typeface getCustomfont() {
        return customfont;
    }

    public void setCustomfont(Typeface customfont) {
        this.customfont = customfont;
    }

    public String getFontname() {
        return fontname;
    }

    public void setFontname(String fontname) {
        this.fontname = fontname;
    }
}

File: app/src/main/java/in/encrust/writersden/FontModel.kt
package `in`.encrust.writersden

import android.graphics.Typeface

class FontModel {
    var customfont: Typeface? = null
    var fontname: String? = null

    constructor() {}

    constructor(customfont: Typeface, fontname: String) {
        this.customfont = customfont
        this.fontname = fontname
    }
}
